aerogear/aerogear-ios-sdk
# feat: Security Check Metrics AEROGEAR-2779

### Motivation

This commit adds security check metrics publishing ability to the security module

JIRA: [https://issues.jboss.org/browse/AEROGEAR-2779]

### Description

Added new publish methods, changed current MetricsData to be of type any instead of a dictionary as security metrics are slightly different to other metrics on the server side currently. 

### Verification steps
This accompanying [PR](https://github.com/aerogear/ios-showcase-template/pull/23) can be used for verification purposes.

* Install mobile core locally
* From the mobile tab, create a metrics service
* Point the app at this metrics service
* Run the showcase application and navigate to the Device trust page
* Metrics should be populated in Grafana
* Change the DeviceTrustService.swift file to the following: https://gist.github.com/StephenCoady/b4d96ec4630decdc17d968bdc0fa3c65 and run the app again to publish all metrics at once as opposed to one at a time.
